cds_ils/cli.py

```diff
@@ -229,6 +229,8 @@ def run_command(command, catch_exceptions=False, verbose=True):
     run_command("vocabulary index opendefinition opendefinition --force")
     # languages
     run_command("vocabulary index languages --force")
+    # countries
+    run_command("vocabulary index languages --force")
```

Comment:
why do we need to run the same command twice?

Reply:
Because it should be `countries` instead! Thank you for spotting this, fixed! :) 👍🏼